ceph_deploy/install.py

```diff
@@ -312,35 +315,15 @@ def uninstall(args):
             use_rhceph=True)
         LOG.info('Distro info: %s %s %s', distro.name, distro.release, distro.codename)
         rlogger = logging.getLogger(hostname)
-        rlogger.info('uninstalling Ceph on %s' % hostname)
-        distro.uninstall(distro)
+        rlogger.info('%s Ceph on %s' % (remove_action, hostname))
+        distro.uninstall(distro, purge)
```

Comment:
Would you be open to changing this line to: `distro.uninstall(distro, purge=purge)`?  `purge` is a kwarg here, and I would prefer to see that called out explicitly rather than it looking like a positional argument.